### ISSUE TITLE
database local postgres and ftp/webdav mode

### ISSUE THREAD

**Issue by wkinoox10** [jsmaskeen/sa-drive#20]
Would it be possible to implement the local postgresql database instead of mongodb? I tested it by just changing the url but the code is all modified to only work with mongodb and I don't understand much about Python to be able to modify it

And would it also be possible to connect in ftp or webdav mode?

thanks

**Comment by jsmaskeen** [comment]
Yes. In the [current release](https://github.com/jsmaskeen/sa-drive/releases/tag/v2.0.2), I've used a local SQLite3 database, and it supports mounting the sadrive as a filesystem.
